### ISSUE TITLE
Invisible column with weird title on place profile (voiceover)

### ISSUE THREAD

**Issue by smcmurtry** [cds-snc/covid-alert-portal#414]
On the "place profile" page in the public exposures portal, there is a third column visible to voiceover titled "contact email blank". Not sure what should be titled but probably not that.

<img width="1792" alt="Screen Shot 2021-03-08 at 2 20 38 PM" src="https://user-images.githubusercontent.com/5498428/110383711-e9cd0f80-8019-11eb-93ac-7f7cc3f8f313.png">
